rootfs/usr/local/bin/shopware-empty-cache/shopware-empty-cache.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# In Shopware 6 Standard the cache is not automatically emptied.
+# In the long run, this can lead to the store installation becoming larger and
+# larger and requiring more and more memory on the server.
+# See: https://docs.shopware.com/en/shopware-6-en/configuration/caches-indexes
+#
+# This script simply clears the cache for every given interval.
+# Interval is every 24 hours.
+
+SHOPWARE_ROOT_DIRECTORY="${SHOPWARE_ROOT_DIRECTORY:-/var/www/html}"
+SHOPWARE_EMPTY_CACHE_INTERVAL="${SHOPWARE_EMPTY_CACHE_INTERVAL:-86400}"
+
+if [ ${SHOPWARE_AUTOMATICALLY_EMPTY_CACHE_ENABLED} == "false" ]; then
+  exit 0; # Not enabled, so exit without error
+fi
+
+echo "Clearing Shopware cache every ${SHOPWARE_EMPTY_CACHE_INTERVAL} seconds"
+
+while [ ${SHOPWARE_AUTOMATICALLY_EMPTY_CACHE_ENABLED} == "true" ]; do
+  echo "Clearing Shopware cache"
```

Comment:
Maybe add the time before the output to see when it happend

Reply:
Yeah having a timestamp is for sure useful. What is currently the standard in this project? 

Because there are multiple types of things going on, there are multiple output formats going on. (see below) I would normally capture the container's stdout into some other process, for example the system's journal. Then I have a unified way of getting a representative timestamp for a message.

Example of current output:
```
INFO[2022-01-14T16:12:04Z] load configuration from file                  file=/etc/supervisord.conf
time="2022-01-14T16:12:04Z" level=info msg="create process:shopware-empty-cache"
time="2022-01-14T16:12:04Z" level=info msg="create process:php-fpm"
time="2022-01-14T16:12:04Z" level=info msg="create process:nginx"
time="2022-01-14T16:12:04Z" level=info msg="stop listening"
time="2022-01-14T16:12:04Z" level=info msg="try to start program" program=php-fpm
time="2022-01-14T16:12:04Z" level=info msg="try to start program" program=nginx
time="2022-01-14T16:12:04Z" level=info msg="try to start program" program=shopware-empty-cache
time="2022-01-14T16:12:04Z" level=info msg="success to start program" program=shopware-empty-cache
Clearing shopware cache every 60 seconds
Clearing shopware cache
2022/01/14 16:12:04 [notice] 39#39: using the "epoll" event method
2022/01/14 16:12:04 [notice] 39#39: nginx/1.20.2
2022/01/14 16:12:04 [notice] 39#39: OS: Linux 5.15.13-200.fc35.x86_64
2022/01/14 16:12:04 [notice] 39#39: getrlimit(RLIMIT_NOFILE): 524288:524288
2022/01/14 16:12:04 [notice] 39#39: start worker processes
2022/01/14 16:12:04 [notice] 39#39: start worker process 44
2022/01/14 16:12:04 [notice] 39#39: start worker process 45
2022/01/14 16:12:04 [notice] 39#39: start worker process 46
2022/01/14 16:12:04 [notice] 39#39: start worker process 47

```

Example of journalctl output:

```
Jan 14 17:12:04 hostname conmon[89944]: INFO[2022-01-14T16:12:04Z] load configuration from file                  file=/etc/supervisord.conf
Jan 14 17:12:04 hostname conmon[89944]: time="2022-01-14T16:12:04Z" level=info msg="create process:shopware-empty-cache"
Jan 14 17:12:04 hostname conmon[89944]: time="2022-01-14T16:12:04Z" level=info msg="create process:php-fpm"
Jan 14 17:12:04 hostname conmon[89944]: time="2022-01-14T16:12:04Z" level=info msg="create process:nginx"
Jan 14 17:12:04 hostname conmon[89944]: time="2022-01-14T16:12:04Z" level=info msg="stop listening"
Jan 14 17:12:04 hostname conmon[89944]: time="2022-01-14T16:12:04Z" level=info msg="try to start program" program=php-fpm
Jan 14 17:12:04 hostname conmon[89944]: time="2022-01-14T16:12:04Z" level=info msg="try to start program" program=nginx
Jan 14 17:12:04 hostname conmon[89944]: time="2022-01-14T16:12:04Z" level=info msg="try to start program" program=shopware-empty-cache
Jan 14 17:12:04 hostname conmon[89944]: time="2022-01-14T16:12:04Z" level=info msg="success to start program" program=shopware-empty-cache
Jan 14 17:12:04 hostname conmon[89944]: Clearing shopware cache every 60 seconds
Jan 14 17:12:04 hostname conmon[89944]: Clearing shopware cache
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: using the "epoll" event method
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: nginx/1.20.2
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: OS: Linux 5.15.13-200.fc35.x86_64
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: getrlimit(RLIMIT_NOFILE): 524288:524288
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: start worker processes
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: start worker process 44
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: start worker process 45
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: start worker process 46
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: start worker process 47
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: start worker process 48
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: start worker process 49
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: start worker process 50
Jan 14 17:12:04 hostname conmon[89944]: 2022/01/14 16:12:04 [notice] 39#39: start worker process 51
Jan 14 17:12:04 hostname conmon[89944]: [14-Jan-2022 16:12:04] NOTICE: fpm is running, pid 38
Jan 14 17:12:04 hostname conmon[89944]: [14-Jan-2022 16:12:04] NOTICE: ready to handle connections
Jan 14 17:12:05 hostname conmon[89944]: time="2022-01-14T16:12:05Z" level=info msg="success to start program" program=nginx
Jan 14 17:12:05 hostname conmon[89944]: time="2022-01-14T16:12:05Z" level=info msg="success to start program" program=php-fpm
Jan 14 17:13:04 hostname conmon[89944]: Clearing shopware cache
Jan 14 17:14:04 hostname conmon[89944]: Clearing shopware cache
```